recipes/ckanparams.rb

```diff
@@ -29,14 +29,39 @@
 node.default['datashades']['ckan_web']['google']['analytics_id'] = `aws ssm get-parameter --region "#{node['datashades']['region']}" --name "/config/CKAN/#{node['datashades']['version']}/common/GaId" --query "Parameter.Value" --output text`.strip
```

Comment:
its really looking like we may want a util shell script to sanity test/tell us if we missed any params in the param store for fresh aws instances. (or user error).